### PR TITLE
feat(rebrand): Rebrand the Repo as BindPlane Agent

### PR DESCRIPTION
**Pull request by cpheps**
### Proposed Change
Rebranding the repo as **BindPlane Agent**.

Items that are changed:
- repo name (after PR merge)
- Go modules
- Documentation when referring to the `collector` or `observIQ's Distro for OpenTelemetry Collector`
- Makefile `make collector` -> `make agent`
- Created additional `bindplane-agent` containers for release deliverables
- Installation script text (not any logic)

Items that are **NOT** changed:
- Binaries
- Installation paths, packages, services
  - These were not change to ensure no user issues with upgrades
- Release artifacts (Note: this will change at some point in the near future)

##### Checklist
- [ ] Changes are tested
- [ ] CI has passed

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -241,6 +254,19 @@ dockers:
...
+      - "observiq/bindplan-agent-arm64:{{ .Major }}.{{ .Minor }}.{{ .Patch }}"
+      - "observiq/bindplan-agent-arm64:{{ .Major }}.{{ .Minor }}"
+      - "observiq/bindplan-agent-arm64:{{ .Major }}"
+      - "ghcr.io/observiq/bindplan-agent-arm64:latest"
+      - "ghcr.io/observiq/bindplan-agent-arm64:{{ .Major }}.{{ .Minor }}.{{ .Patch }}"
+      - "ghcr.io/observiq/bindplan-agent-arm64:{{ .Major }}.{{ .Minor }}"
+      - "ghcr.io/observiq/bindplan-agent-arm64:{{ .Major }}"
+      - "us-central1-docker.pkg.dev/observiq-containers/agent/bindplan-agent-arm64:latest"
+      - "us-central1-docker.pkg.dev/observiq-containers/agent/bindplan-agent-arm64:{{ .Major }}.{{ .Minor }}.{{ .Patch }}"
+      - "us-central1-docker.pkg.dev/observiq-containers/agent/bindplan-agent-arm64:{{ .Major }}.{{ .Minor }}"
```
Comment (Mrod1598) [inline .goreleaser.yml:268]: missing an e on all these

File: README.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-# observIQ Distro for OpenTelemetry Collector
+# BindPlane Agent
 
 <center>
 
-[![Action Status](https://github.com/observIQ/observiq-otel-collector/workflows/Build/badge.svg)](https://github.com/observIQ/observiq-otel-collector/actions)
-[![Action Test Status](https://github.com/observIQ/observiq-otel-collector/workflows/Tests/badge.svg)](https://github.com/observIQ/observiq-otel-collector/actions)
+[![Action Status](https://github.com/observIQ/bindplane-agent/workflows/Build/badge.svg)](https://github.com/observIQ/bindplane-agent/actions)
+[![Action Test Status](https://github.com/observIQ/bindplane-agent/workflows/Tests/badge.svg)](https://github.com/observIQ/bindplane-agent/actions)
 [![Go Report Card](https://goreportcard.com/badge/github.com/observIQ/observiq-otel-collector)](https://goreportcard.com/report/github.com/observIQ/observiq-otel-collector)
```
Comment (Mrod1598) [inline README.md:10]: needs to be updated here?

**Review by Mrod1598 — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -34,19 +34,19 @@ Tested, verified, and supported by observIQ
...
+To install the BindPlane Agent on Windows run the Powershell command below to install the MSI with no UI.
 ```pwsh
-msiexec /i "https://github.com/observIQ/observiq-otel-collector/releases/latest/download/observiq-otel-collector.msi" /quiet
+msiexec /i "https://github.com/observIQ/bindplane-agent/releases/latest/download/observiq-otel-collector.msi" /quiet
 ```
 
-Alternately, for an interactive installation [download the latest MSI](https://github.com/observIQ/observiq-otel-collector/releases/latest).
+Alternately, for an interactive installation [download the latest MSI](https://github.com/observIQ/bindplane-agent/releases/latest).
 
 After downloading the MSI, simply double click it to open the installation wizard. Follow the instructions to configure and install the collector.
```
Comment (Mrod1598) [inline README.md:51]: agent

File: config/google_cloud_exporter/cadvisor/README.md [hunk 4]
```diff
@@ -36,8 +36,8 @@ services:
     - /sys:/sys:ro
     - /var/lib/docker/:/var/lib/docker:ro
 
-  collector:
-    image: observiq/observiq-otel-collector:1.1.0
+  agent:
+    image: observiq/bindplane-agent:1.30.0
     restart: always
     container_name: observiq-otel-collector
```
Comment (Mrod1598) [inline config/google_cloud_exporter/cadvisor/README.md:42]: container name staying the same?

**Review by Mrod1598 — APPROVED** [review]
few more small things but other than that LGTM

File: README.md (hunk 3 above)
Comment (cpheps) [reply README.md:51]: Saw a few more of these and replaced them.

File: config/google_cloud_exporter/cadvisor/README.md (hunk 4 above)
Comment (cpheps) [reply config/google_cloud_exporter/cadvisor/README.md:42]: No changing it

**Review by jsirianni — APPROVED** [review]

File: windows/templates/CustomExitDialog.wxs [hunk 5]
```diff
@@ -10,19 +10,19 @@
...
+To restart agent the service, run:
     Restart-Service -Name observiq-otel-collector" >
                     <Condition Action="show">NOT Installed</Condition>
                 </Control>
                 <Control Id="UninstallDescription" Hidden="yes" Type="Text" X="135" Y="70" Width="220" Height="40" Transparent="yes" NoPrefix="yes" Text="!(loc.ExitDialogDescription)" >
                     <Condition Action="show">Installed</Condition>
                 </Control>
                 <Control Id="DocLink" Hidden="yes" Type="Hyperlink" X="135" Y="200" Width="220" Height="20" Transparent="yes">
-                    <Text><![CDATA[<a href="https://github.com/observIQ/observiq-otel-collector/blob/main/docs/installation-windows.md#configuring-the-collector">Click here for online documentation on configuration.</a>]]></Text>
+                    <Text><![CDATA[<a href="https://github.com/observIQ/bindplane-agent/blob/main/docs/installation-windows.md#configuring-the-collector">Click here for online documentation on configuration.</a>]]></Text>
```
Comment (antonblock) [inline windows/templates/CustomExitDialog.wxs:25]:
```suggestion
                    <Text><![CDATA[<a href="https://github.com/observIQ/bindplane-agent/blob/main/docs/installation-windows.md#configuring-the-agent">Click here for online documentation on configuration.</a>]]></Text>
```

**Review by antonblock — APPROVED** [review]
I think the heading in that link needs to be updated, otherwise looks good

File: windows/templates/CustomExitDialog.wxs (hunk 5 above)
Comment (cpheps) [reply windows/templates/CustomExitDialog.wxs:25]: Good catch

**Pull request merged**